packages/xod-arduino-deploy/src/boardsParser.js

```diff
@@ -13,7 +13,19 @@ import * as Utils from './utils';
 /** Parses Arduino's `.txt` definition file. Like `boards.txt` or `platform.txt` */
 // :: String -> Object
 const parseTxtConfig = R.compose(
-  R.reduce((txtConfig, [tokens, value]) => R.assocPath(tokens, value, txtConfig), {}),
+  R.reduce(
+    (acc, [tokens, value]) => R.compose(
+      R.assocPath(R.__, value, acc),
+      R.when(
+        R.both(
+          R.propEq(1, 'menu'),
+          R.compose(R.equals(4), R.length)
+        ),
+        R.append('cpuName')
+      )
+    )(tokens),
+    {}
+  ),
```

Comment:
I only understood why all of this is happening because you showed me the example yesterday 🙈
A docstring with some example/explanation would be really nice here.